apps/system/js/input_window_manager.js

```diff
@@ -237,9 +240,15 @@
         break;
       case 'lockscreen-appopened':
       case 'sheets-gesture-begin':
+      case 'cardviewbeforeshow':
         if (this._hasActiveInputApp()) {
           // Instead of hideInputWindow(), we should removeFocus() here.
           // (and removing the focus cause Gecko to ask us to hideInputWindow())
+
+          // We need to blur the app to prevent the keyboard from refocusing
+          // right away
+          var app = Service.currentApp;
+          app && app.blur();
```

Comment:
yes otherwise we get another `inputmethod-contextchange` right away :/
I will file a bug and reference it here